osc/iterm2.go

```diff
@@ -19,11 +22,46 @@ var st = "\007"
 var cellSizeOnce sync.Once
 var cellWidth, cellHeight float64
 
+var sixelEnabled = false
+
 func init() {
 	if os.Getenv("TERM") == "screen" {
 		ecsi = "\033Ptmux;\033" + ecsi
 		st += "\033\\"
 	}
+	sixelEnabled = checkSixel()
```

Comment:
It's better if perform this check on demand. This way, if we detect iTerm2 we don't have to check for Sixel which is one less opportunity to mess-up with the term. We can still store the result so we don't have to perform the check every time.

To be more future proof, I would store the terminal type instead of a Boolean.